src/arviz_plots/visuals/__init__.py

```diff
@@ -13,6 +13,21 @@
 from arviz_base.labels import BaseLabeller
 
 
+def plot_hist(da, target, backend, **kwargs):
```

Comment:
I would use `hist` only to "reserve" the `plot_xyz` for batteries included functions, otherwise it might get confusing.